src/test/java/com/exasol/adapter/metadata/DataTypeTest.java

```diff
@@ -46,16 +46,30 @@ void createChar() {
 
     @Test
     void createTimestampWithLocalTimezone() {
-        final DataType dataType = DataType.createTimestamp(true);
+        final DataType dataType = DataType.createTimestamp(true, 3);
         assertAll(() -> assertTrue(dataType.isWithLocalTimezone()),
-                () -> assertThat(dataType.toString(), equalTo("TIMESTAMP WITH LOCAL TIME ZONE")));
+                () -> assertThat(dataType.toString(), equalTo("TIMESTAMP(3) WITH LOCAL TIME ZONE")));
+    }
```

Comment:
We are still waiting for word from the core whether removing the precision in case of the standard precision of 3 is a valid complexity reduction when it comes to backward compatibility.